### DISCUSSION TITLE
ShapeMenuItem

### DISCUSSION THREAD

**Discussion by dpomt** [OrchardCMS/OrchardCore#14661] [General]
Orchard 1.x has a ShapeMenu item. 

![image](https://github.com/OrchardCMS/OrchardCore/assets/1379660/07a4dcd9-0046-416d-b9ba-ab01dcfe897b)

![image](https://github.com/OrchardCMS/OrchardCore/assets/1379660/09386e91-cb48-43de-852e-67cc78e266b8)

![image](https://github.com/OrchardCMS/OrchardCore/assets/1379660/cfe0f837-6764-43b5-b53d-9f23c065ff41)

Would great to have this in OC too.

**Comment by sebastienros** [comment]
Sure it exists in Orchard Core. Menu items are extensible, check for the types that are available. There are content items too so you can customize their templates (content type shapes). Even from the admin.

**Comment by dpomt** [comment]
Thanks @sebastienros for the feedback.
Was not aware that is sufficient to create a new contenttype with stereotype "MenuItem" to get a new MenuItem type.
Works perfekt, thanks.
